models/user_test.go

```diff
@@ -318,6 +319,77 @@ func TestFindAllUsers(t *testing.T) {
 	}
 }
 
+// TestUserById tests that you can get a user by its ID.
+//
+func TestUserUpdate(t *testing.T) {
+	var c aetest.Context
+	var err error
+	options := aetest.Options{StronglyConsistentDatastore: true}
+
+	if c, err = aetest.NewContext(&options); err != nil {
+		t.Fatal(err)
+	}
+	defer c.Close()
+
+	/*Test data: saved user*/
+	var user *User
+	if user, err = CreateUser(c, "foo@bar.com", "john.snow", "john snow", "crow", false, ""); err != nil {
+		t.Errorf("Error: %v", err)
+	}
+
+	/*Test data: non saved user*/
+	nonSavedUser := User{
```

Comment:
The context `c` is needed to update the user later on. But this function can be used in other tests where the context is no longer needed after the creation of a user.